packages/pixeloven/cli-core/src/commands/copy.ts

```diff
@@ -14,6 +14,13 @@ export default {
                 });
                 print.success(`Successfully copied ico files to dist`);
                 break;
+            case "images":
```

Comment:
@ductiletoaster I started out replacing the existing svg task but that would make a breaking change so I added this one instead. Lemme know if you want to remove the svg task and let the project update to a major release